### ISSUE TITLE
Remove require of encoding-indexes.js

### ISSUE THREAD

**Issue by arv** [inexorabletash/text-encoding#44]
Right now `encoding-indexes.js` is always required in a node like environment (such as browserify).

This was done in a85ce6ff385b71a9cafdad46cb6965ff3bd2de36

It would be better if `encoding-indexes.js` was never implicitly included.

**Comment by inexorabletash** [comment]
Suggestions (or PRs) appreciated. I want to keep things testable out of the box (i.e. runnable via gh-pages) but otherwise I'm amenable to pretty much anything.

**Comment by cgwyllie** [comment]
Something with this that would allow selective inclusion of different encodings would be good for us. We are only interested in UTF-8 and build using webpack, but we end up with all encodings getting bundled.

Not sure what the best option might be..

**Comment by inexorabletash** [comment]
Again, happy to do this but I am unfamiliar with these environments and so guidance is needed.

**Comment by surfjedi** [comment]
Yes I am experiencing the same thing - using web pack to build and don't want indexes.. great polyfill tho
